src/lsp/client/client.ts

```diff
@@ -60,8 +60,8 @@ export function activate(
 
     // Create the language client and start the client.
     client = new LanguageClient(
-        'LSP Mobile',
-        'LSP Mobile Client',
+        'Salesforce LSP Mobile',
+        'Salesforce LSP Mobile Client',
```

Comment:
Updated name so it is displayed as a unique entry in VSCode.
![image](https://github.com/user-attachments/assets/6b55c3b9-68ae-43df-ba9d-f8a59ad1f698)

Reply:
nitpick:   should we name it to 'Salesforce LSP mobile'? I make lsp server output some log like 
<img width="856" alt="image" src="https://github.com/user-attachments/assets/bc0a7901-3f0c-4641-aeed-5f5dbf773af7">
it's in the Salesforce LSP Mobile Client console stream.

Reply:
@ben-zhang-at-salesforce Double checking. Your screenshot is chopped off but that's where you want the name to be "Salesforce LSP mobile", right?

Reply:
Maybe `Salesforce Mobile Language Server` is a better fit ?